recipes/gsoap/all/conandata.yml

```diff
@@ -1,21 +1,12 @@
 sources:
-  "2.8.117":
+  "2.8.129":
     url:
-      - "https://sourceforge.net/projects/gsoap2/files/gsoap_2.8.117.zip/download"
-      - "https://c3i.jfrog.io/artifactory/cci-sources-backup/sources/gsoap/gsoap_2.8.117.zip"
+      - "https://downloads.sourceforge.net/project/gsoap2/gsoap_2.8.129.zip"
+      - "https://c3i.jfrog.io/artifactory/cci-sources-backup/sources/gsoap/gsoap_2.8.129.zip"
```

Comment:
This file does not exist. They are added here because this library is prone to getting its sources deleted. We should check int how to handle this onw that the backup source feature is working in CCI, give us a few moments please!

Reply:
Moving forward with using our mirror for now, as the clients do not have backup sources enabled by default